gooey/gui/model.py

```diff
@@ -207,8 +208,18 @@ def skipping_config(self):
     return self.build_spec['manual_start']
 
   def is_required_section_complete(self):
-    completed_values = filter(None, [arg.value for arg in self.required_args])
-    return len(self.required_args) == len(completed_values)
+    error_found = False
+    for arg in self.required_args:
+      widget = arg.widget_instance.widget_pack.widget

```

Comment:
You should call `widget.Refresh()` on everything. When there are multiple required fields, I'm  only seeing the first one change colors until I mouse over the rest.

Reply:
I can't reproduce this. But I've done as you requested, if you can check that it works as intended.